src/browser/components/common/requests.js

```diff
@@ -1,25 +1,52 @@
 const post = (url, body) => {
    return fetch(url, {
      method:'POST',
+     headers: {'Content-Type': 'text/plain'},
      mode: 'cors',
      body: JSON.stringify(body),
      credentials: 'same-origin'
    })
  }
 
- const get = (url, params={}) => {
+ const getUserName = (url, params={}) => {
    return fetch(url, {
      method:'GET',
-     header: 'Content-Type, application/JSON',
+     headers: {'Content-Type': 'application/json'},
      mode: 'cors',
      credentials: 'same-origin'
    }).then(result => {
      return result.json()
    })
  }
 
- // add put request here
+ const getDatabaseQuestions = (url, params={}) => {
```

Comment:
You shouldn't be creating multiple functions for different routes. Instead there should be generic functions like `get`, `post`, `put` & `delete` which accept a `url` as a parameter.